app/models/timer.rb

```diff
@@ -0,0 +1,31 @@
+class Timer < ActiveRecord::Base
+
+  belongs_to :player
+  belongs_to :game
+
+  def initialize
+    @start_time = nil
+    @stop_time = nil
+    @running = false
+    @time_left = 500
+  end
+
+  def start
+    @time_left 
```

Comment:
This line sets how much time left on the timer as you go through your turns. If you start with 5 minutes, take a turn, and now have 3 minutes, @time_left should keep track of how much time you have left from each turn during that game. At least that is what I hope it does.